ras_backstage/controllers/collection_exercise_controller.py

```diff
@@ -0,0 +1,27 @@
+import json
+import logging
+
+from structlog import wrap_logger
+
+from ras_backstage import app
+from ras_backstage.common.requests_handler import request_handler
+from ras_backstage.exception.exceptions import ApiError
+
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def get_collection_exercises_by_survey(survey_id):
+    logger.debug('Retrieving collection exercises', survey_id=survey_id)
+    url = f'{app.config["RM_COLLECTION_EXERCISE_SERVICE"]}collectionexercises/survey/{survey_id}'
```

Comment:
Shouldn't we be consistent with these in that we either use .format or this method throughout the whole app?

Reply:
Consistency is preferred but as f-strings are not intended to replace str.format but provide an alternative short-hand we can have a mix (especially for different use-cases).

"[This PEP does not propose to remove or deprecate any of the existing string formatting mechanisms](https://www.python.org/dev/peps/pep-0498/#abstract)."